### PR TITLE
[372] - handle approve comment endpoint

### PR DESCRIPTION
**Pull request by DaDer**
Trello: https://trello.com/c/7AqKUAv5/372-5-as-a-user-i-want-to-approve-and-delete-comments

### CODE REVIEW

File: CreatubblesAPIClient.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2987,10 +3003,10 @@
 			isa = PBXSourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				3F7AD4461EB70F540044F6D6 /* BuildFile in Sources */,
-				B225DF721EB31F2200AB6F4C /* BuildFile in Sources */,
-				3F7AD4441EB70B6C0044F6D6 /* BuildFile in Sources */,
-				B225DF6E1EB31F2200AB6F4C /* BuildFile in Sources */,
+				3F7AD4461EB70F540044F6D6 /* (null) in Sources */,
```
Comment (sebastianewak) [inline CreatubblesAPIClient.xcodeproj/project.pbxproj:3006]: Can these be removed?

File: CreatubblesAPIClient.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -924,7 +932,7 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				DCB6AE268E81388D94B0CE09 /* BuildFile in Frameworks */,
+				DCB6AE268E81388D94B0CE09 /* (null) in Frameworks */,
```
Comment (sebastianewak) [inline CreatubblesAPIClient.xcodeproj/project.pbxproj:935]: Can this be removed?

File: CreatubblesAPIClient.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -404,21 +408,21 @@
 		21CC9E401DF834A100C6538C /* SearchTag.swift in Sources */ = {isa = PBXBuildFile; fileRef = 21CC9E3F1DF834A100C6538C /* SearchTag.swift */; };
 		21CC9E431DF8361600C6538C /* SearchTagsFetchRequest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 21CC9E421DF8361600C6538C /* SearchTagsFetchRequest.swift */; };
 		21CC9E451DF8362C00C6538C /* SearchTagsFetchResponseHandler.swift in Sources */ = {isa = PBXBuildFile; fileRef = 21CC9E441DF8362C00C6538C /* SearchTagsFetchResponseHandler.swift */; };
-		3F7AD4441EB70B6C0044F6D6 /* BuildFile in Sources */ = {isa = PBXBuildFile; };
-		3F7AD4461EB70F540044F6D6 /* BuildFile in Sources */ = {isa = PBXBuildFile; };
+		3F7AD4441EB70B6C0044F6D6 /* (null) in Sources */ = {isa = PBXBuildFile; };
```
Comment (sebastianewak) [inline CreatubblesAPIClient.xcodeproj/project.pbxproj:411]: Have some files been removed from the project?

**Review by sebastianewak — APPROVED** [review]
Code: +1
Func: n\a

File: CreatubblesAPIClientIntegrationTests/Spec/Comment/DeclineCommentResponseHandlerSpec.swift [hunk 4]
```diff
@@ -1,5 +1,5 @@
 //
-//  CreationUploadMapper.swift
+//  DeclineCommentResponseHandlerSpec.swift
```
Comment (mmiedlarz) [inline CreatubblesAPIClientIntegrationTests/Spec/Comment/DeclineCommentResponseHandlerSpec.swift:2]: 😍

**Review by mmiedlarz — APPROVED** [review]
Code 👍 
Func N/A

**Pull request merged**